### DISCUSSION TITLE
Can I get a good first issue?

### DISCUSSION THREAD

**Discussion by inhan05053** [diegomura/react-pdf#1615] [Ideas]
Hello @jeetiss !

I really appreciate your help.
Since we don't know yet, can you leave some good first issues that we can solve?
We are going to try as hard as we can.

**Comment by jeetiss** [comment]
Hello folks.
Thanks again for your interest

You would have to start with a conversation like this instead of rushing a random tricky issue. react-pdf is difficult to contribute, but with your interest, I hope we can make it easier.

There is one easy-to-fix problem:
- https://github.com/diegomura/react-pdf/issues/1616

just pick the description from readme and add it to package json config. it helps a lot

I'm going to find something else, maybe you have some preferences? How many issues are you need?)

**Reply by inhan05053** [reply]
Thank you for preparing a good first issue.

I don't know if it's possible to contribute, but please leave an additional good first issue.

thank you!

**Comment by jeetiss** [comment]
let go next!

I think you experienced problems with committing to repository when you had worked on #1624 
and here the reason 

- https://github.com/diegomura/react-pdf/issues/1625

I have problems with this too, so you can upgrade and check how husky and lint-staged work in [other repos](https://typicode.github.io/husky/#/?id=used-by) and fix this behaviour.

**Reply by YKIM-98** [reply]
Thank you so much for your interest!! @jeetiss 
We'll do the next mission and send the pr. 
Thanks!!

**Comment by jeetiss** [comment]
- https://github.com/diegomura/react-pdf/issues/1606

you can fix this one. it should be easy, but have the pitfall

this is the third issue on month

- https://github.com/diegomura/react-pdf/pull/1577
- https://github.com/diegomura/react-pdf/pull/1557

could you please check is it possible to find all scope problems automatically with eslint or else?
